pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1521,20 +1521,36 @@ public void removeBacklogQuota(@Suspended final AsyncResponse asyncResponse,
         internalRemoveBacklogQuota(asyncResponse, backlogQuotaType);
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/{topic}/messageTTLApplied")
```

Comment:
I am curious about why not add a query Params in the original API rather introduce a new API?

Reply:
+1

Reply:
If we reuse the original GET method, compatibility is a problem. I have tried the following paths, but none of them are compatible:
```
/{tenant}/{namespace}/{topic}/{applied}/inactiveTopicPolicies
/{tenant}/{namespace}/{topic}/inactiveTopicPolicies/{applied}
```

Reply:
I mean like this: 
```
/{tenant}/{namespace}/{topic}/inactiveTopicPolicies?applied=true
```
By default, it can be false

Reply:
This way is different from the previous, but I think it is OK. what do you think? @codelipenghui

Reply:
We should use the query parameter instead of a path parameter.